Detail how to create and patch your own nand from scratch using the rootfs from the ipsw

zoe-vb opened the pull request:
Figured out all the changes you had made to the original filesystem-writable.dmg and documented them and how to apply them.

zoe-vb committed:
Create README.md

zoe-vb committed:
Create changes.md

zoe-vb committed:
Update README.md

zoe-vb committed:
Update changes.md

Commit:
Add resources folder

Commit:
whoops... forgot to add xpwn

zoe-vb committed:
Update README.md

devos50 commented:
Very impressive work! Thank you for reverse engineering my changes 😅 . Please let me know when this PR is ready and I'll merge it.

For the IT2G, I haven't modified the filesystem (yet) but I'll document the changes as I'll proceed!

zoe-vb committed:
Update changes.md

zoe-vb committed:
Update changes.md

Completed guide, now fixed any errors (I tested it again to make sure it was perfect)

devos50 reviewed (approved):
Thank you! 👏
